package.json

```diff
@@ -4,22 +4,23 @@
   "private": false,
   "dependencies": {
     "@babel/runtime": "^7.7.2",
-    "@patternfly/patternfly": "4.10.31",
-    "@patternfly/react-core": "4.23.1",
-    "@patternfly/react-icons": "4.3.5",
-    "@patternfly/react-styles": "4.3.4",
-    "@patternfly/react-table": "4.5.7",
+    "@patternfly/patternfly": "4.42.2",
+    "@patternfly/react-core": "4.57.2",
+    "@patternfly/react-icons": "4.7.6",
+    "@patternfly/react-styles": "4.7.5",
+    "@patternfly/react-table": "4.17.9",
     "@redhat-cloud-services/frontend-components": "2.3.13",
     "@redhat-cloud-services/frontend-components-notifications": "2.1.2",
     "@redhat-cloud-services/frontend-components-utilities": "2.1.3",
-    "@redhat-cloud-services/insights-common-typescript": "1.0.5",
+    "@redhat-cloud-services/insights-common-typescript": "3.0.1",
     "@redhat-cloud-services/rbac-client": "1.0.71",
     "axios": "^0.19.2",
     "camelcase": "5.3.1",
     "classnames": "^2.2.6",
     "csstips": "^1.2.0",
     "csx": "^10.0.1",
     "date-fns": "^2.9.0",
+    "emotion": "^10.0.27",
```

Comment:
professional preference, lockin down all deps to exact versions, less surprises 🤷 🏆 :trollface: 🤗

Reply:
Actually this shouldn't even be here. Had to do some imports when testing with patternfly/react-core. I'll just remove this.

Reply:
> lockin down all deps to exact versions

Ah, I never got to finish locking all the dependencies, that's something I have to do.